src/components/homepage/GetStartedTerminal.tsx

```diff
@@ -1,45 +1,42 @@
+import clsx from "clsx";
 import React, { FunctionComponent } from "react";
-import 'react-animated-term/dist/react-animated-term.css';
-import "./GetStartedTerminal.overrides.css";
-import Terminal from 'react-animated-term';
+import styles from "./GetStartedTerminal.module.css";
 
 
-const spinner = ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'];
-const cmdStep = (title: string) => ({
-  text: title,
-  cmd: true,
-  delay: 10
-})
-
-const spinningStep = (title: string, repeatCount = 0, frameDelay = 10) => ({
-  text: `✔ ${title}`,
-  cmd: false,
-  repeat: repeatCount > 0,
-  repeatCount: repeatCount,
-  frames: spinner.map((spinnerItem) => ({
-    text: `${spinnerItem} ${title}...`,
-    delay: frameDelay
-  }))
-})
-
-const termLines = [
-  cmdStep("capact env create kind"),
-  spinningStep("Creating cluster `kind-dev-capact`"),
-  cmdStep("capact install"),
-  spinningStep("Installing Capact on cluster"),
-  cmdStep("capact login https://gateway.capact.local -u user -p pass"),
+const terminalData = [
+  {
+    text: "# Create local kind cluster",
+    dimmed: true,
+  },
   {
-    text: '✔ Login succeeded',
-    delay: 10
+    text: "capact env create kind",
   },
-  cmdStep("# You're all set! Enjoy using Capact!"),
+  {
+    text: "# Install Capact",
+    dimmed: true,
+  },
+  {
+    text: "capact install",
+  },
+  {
+    text: "# Log in to the Capact Gateway",
+    dimmed: true,
+  },
+  {
+    text: "capact login https://gateway.capact.local -u user -p pass"
```

Comment:
**suggestion**
Maybe we should also update our installation scripts to use by default `user` and `pass`. This will make it even more ready to copy-paste scenario

Reply:
Good idea. I updated the login command to use the interactive mode.